test/installation/node/install from source without V8 snapshot for ARM

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+NVM_TEST_VERSION=v0.10.7
+
+# Remove the stuff we're clobbering.
+[ -e ../../../$NVM_TEST_VERSION ] && rm -R ../../../$NVM_TEST_VERSION
+
+# Fake ARM arch
+nvm_get_arch() {
+  echo "armv7l"
+}
+
+# Install from source
+nvm install -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+
+# Use new version
+nvm use $NVM_TEST_VERSION
```

Comment:
This is done automatically by `nvm install`, so either it should be removed, or a `|| die` should be added in case this line fails.